content/organization/events/online/index.en.md

```diff
@@ -56,14 +57,22 @@ to the calendly booking, e.g., book for 5:45pm and tell your meetup
 participants that it starts at 6pm. **Please be punctual so that the
 Host is there to manage the meetup before others show up.**
 
-You can claim the Host role after you joined the meeting by clicking on
+:warning: You can [claim the Host](#how-to-claim-host-controls-using-the-host-key) role after you joined the meeting by clicking on
 participants and then on "claim host" in the right-hand corner. You will
-need to enter the host key which you'll get from us prior to the
-meetup.
+need to enter the host key which you'll get from \#online\_meetups channel on the 
+R-Ladies Organizer Slack.
 
 :wave: **Please remember to _end_ the meeting, not just _leave_ the meeting** 
 so that the next organizer can access the room for their meetup.
 
+:jigsaw: In short, you will need **two** passwords in order to start your meeting on
+the right foot.
+
+1. **Password** to get into the scheduled zoom booking (sent via e-mail and embedded in 
```

Comment:
ooooh I see. I'd suggest using password in all places, not passcode.

Reply:
reorganised the text to make this distinction earlier, and also always refer to it as passcode.